Sources/SwiftDiscord/Channel/DiscordMessage.swift

```diff
@@ -328,7 +351,14 @@ public struct DiscordEmbed : JSONAble {
         public init(text: String?, iconUrl: URL?) {
             self.text = text
             self.iconUrl = iconUrl
-            self.proxyUrl = nil
+            self.proxyIconUrl = nil
+        }
+
+        /// For testing
+        internal init(text: String?, iconURL: URL?, proxyURL: URL?) {
```

Comment:
Yep, was having too much fun with the fact that Xcode 9 finally got refactoring and forgot to change the other variables named the same thing :)